From a76c7d67ab2a819f4d40e7538218e572f023b7a1 Mon Sep 17 00:00:00 2001
From: Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
Date: Tue, 12 May 2026 22:59:09 +0200
Subject: [PATCH] add 'agian' to the message

---
 src/hello.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hello.c b/src/hello.c
index 453962f..59af4c7 100644
--- a/src/hello.c
+++ b/src/hello.c
@@ -57,7 +57,7 @@ main (int argc, char *argv[])
 #endif
 
   /* Having initialized gettext, get the default message. */
-  greeting_msg = _("Hello, world!");
+  greeting_msg = _("Hello, world again!");
 
   /* Even exiting has subtleties.  On exit, if any writes failed, change
      the exit status.  The /dev/full device on GNU/Linux can be used for
